plugins/scaffolder-backend-module-azure/package.json

```diff
@@ -6,13 +6,26 @@
   "types": "src/index.ts",
   "license": "Apache-2.0",
   "publishConfig": {
-    "access": "public",
-    "main": "dist/index.cjs.js",
-    "types": "dist/index.d.ts"
+    "access": "public"
   },
   "backstage": {
     "role": "backend-plugin-module"
   },
+  "exports": {
+    ".": "./src/index.ts",
+    "./alpha": "./src/alpha.ts",
```

Comment:
Hmm, I feel there's likely to be a higher impact on users by shipping this as alpha at this point tbh. It means that we're going to need an alpha -> stable migration at some point in the future. If we ship this straight into stable for these modules then there won't be any future tweaks unless we need to make some breaking changes, and even then it's likely just gonna be that versions need to be kept in sync.